.github/workflows/icfy-stats.yml

```diff
@@ -0,0 +1,53 @@
+name: Calculate ICFY stats
+
+on: [push]
+
+jobs:
+  build:
+    name: Build ICFY stats
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '>=12.15.0'
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Fetch git history
+      run: git fetch --prune --unshallow
+    - name: Install dependencies
+      env:
+        PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: true
+      run: npm ci
+    - name: Build ICFY stats
+      env:
+        NODE_ENV: production
+        BROWSERSLIST_ENV: defaults
+        WORKERS: 2
+      run: npm run analyze-icfy
+    - run: mkdir icfy-stats && mv client/{chart,stats}.json icfy-stats
+    - uses: actions/upload-artifact@v1
+      with:
+        name: icfy
+        path: icfy-stats
```

Comment:
Can we publish the artifacts as two plain files, `stats.json` and `chart.json`? That would be much more convenient for the ICFY client. It now downloads the files directly into memory. Now, it would need to unpack the ZIP to some tmp directory, read the JSON files from there, cleanup at the end...

Reply:
This uses https://github.com/actions/upload-artifact. Docs make it sound like it will always be a zip:

> You must upload artifacts during a workflow run. GitHub provides two actions that you can use to upload and download build artifacts. Files uploaded to a workflow run are archived using the `.zip` format. For more information, see the actions/upload-artifact and download-artifact actions.

https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts

Reply:
Doesn't seem like it. https://github.com/actions/upload-artifact

It wants a path to a directory and a name

Reply:
not sure it helps, but zips can be used as streams with https://www.npmjs.com/package/node-stream-zip

Reply:
`node-stream-zip` can only read from files 🙁 I need something that can read from a generic stream. Will continue looking for another package.

Reply:
Found https://www.npmjs.com/package/unzipper which works great so far :+1: